### PR TITLE
[TUTORIAL]TFLite QNN Tutorial

### PR DESCRIPTION
**Pull request by siju-samuel**
QNN Tutorial series on tflite. Continuation of PRs #5321(PyTorch) & #5362(MxNet)
This PR can be merged after #5362

@anijain2305 @masahi Please help to review this PR.

### CODE REVIEW

File: tutorials/frontend/deploy_prequantized_tflite.py [hunk 1]
```diff
@@ -0,0 +1,244 @@
...
+#   In this case, TVM utilizes the fastest available 8 bit instructions for the given target.
+#   This includes support for the VNNI 8 bit dot product instruction (CascadeLake or newer).
+#   For EC2 C5.12x large instance, TVM latency for this tutorial is ~2 ms.
+#
+#   Moreover, the following general tips for CPU performance equally applies:
+#
+#    * Set the environment variable TVM_NUM_THREADS to the number of physical cores
+#    * Choose the best target for your hardware, such as "llvm -mcpu=skylake-avx512" or
+#      "llvm -mcpu=cascadelake" (more CPUs with AVX512 would come in the future)
+#    * Perform autotuning - `Auto-tuning a convolution network for x86 CPU
```
Comment (anijain2305) [inline tutorials/frontend/deploy_prequantized_tflite.py:243]: Can you please add a comment about ARM tuning here? It is more likely that TVM user interested in this tutorial is looking to run on ARM device.

**Comment by siju-samuel** [comment]
@anijain2305 @masahi Please help to review and merge this PR. TIA

**Comment by masahi** [comment]
Thanks @siju-samuel @anijain2305

**Comment by tqchen** [comment]
FYI, this PR has a problem that was not catched by the CI(due to another problem in sphinx gallery), see https://github.com/apache/incubator-tvm/pull/5641.

We will enhance the CI checker, but please also make sure also to run the tutorial in the ci env :)

**Pull request merged**